pkg/arvo/lib/bip/b158.hoon

```diff
@@ -86,39 +88,44 @@
 ::   write appends to the back
 ::
 ++  str
+  ~%  %str  ..params  ~
   |%
   ++  read-bit
+    ~/  %read-bit
```

Comment:
Since this change can be tested in isolation, I would be happy to approve this in its own PR, if we don't want to delay this for testing all the style changes.